wremnants/datasets/datagroups.py

```diff
@@ -216,15 +217,18 @@ def set_histselectors(
         scale = 1
         if mode == "extended1D":
             fakeselector = sel.FakeSelector1DExtendedABCD
-            scale = 1/1.15
```

Comment:
Ok, there was an option to override this normalisation in setupCombine.py, --forceGlobalScaleFakes, which might be removed at this point, or renamed and translated into a new option that applies any desired scaling. It might be already possible to do such scaling through other exiting options, though, through actions.
Depending on what is done, it might be worth considering having an even more general option to apply arbitrary normalisation scaling for each selected process (it came up also for the DY background to test the veto)

Reply:
OK I reverted this change